src/utils.ts

```diff
@@ -67,6 +77,11 @@ export function nodeType(obj: any): SchemaObjectType | undefined {
     return "array";
   }
 
+  // file
+  if (obj.type === "file") {
+    return "unknown";
```

Comment:
At least for now, we’ll return `unknown` for a file type. Because we’re not sure where this is running—server or browser—it would seem risky to simply assume a `Buffer` or `string` type for a file. Instead, we’ll rely on the implementer to specify how to deal with the file.